src/vsphere_cpi/lib/cloud/vsphere/attach_tag_to_vm.rb

```diff
@@ -0,0 +1,134 @@
+require 'cloud/vsphere/logger'
+require 'vsphere-automation-cis'
+require 'vsphere-automation-vcenter'
+
+module  VSphereCloud
+  module TaggingTag
+    class AttachTagToVm
+      include Logger
+
+      def initialize(api_client)
+        @api_client = api_client
+      end
+
+      private def tag_assoc_info
+        @tag_assoc_info ||= VSphereAutomation::CIS::CisTaggingTagAssociationAttach.new
+      end
+
+      private def multi_tag_assoc_info
+        @multi_tag_assoc_info ||= VSphereAutomation::CIS::CisTaggingTagAssociationAttachMultipleTagsToObject.new
+      end
+
+      private def tag_assoc_api
+        @tag_assoc_api ||= VSphereAutomation::CIS::TaggingTagAssociationApi.new(@api_client)
+      end
+
+      private def tagging_tag_api_instance
```

Comment:
Why is this `_instance` while the one above is not. Use consistent naming.

Reply:
Removed.